db/migrations/0003_add_frames_table.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE frames (
```

Comment:
Yeah I just thought of that as well. But then we need to generate the uuids on the client side, which seems kinda sketch for a primary key lol. I think its fine with hashes